### PR TITLE
Moped Editor | Update project list #4708

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/4708

### CODE REVIEW

File: moped-editor/src/views/projects/Toolbar.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const Toolbar = ({ change, className, ...rest }) => {
                     variant="contained"
                     component={RouterLink}
                     to={"/moped/projects/new"}
+                    startIcon={<Icon>add_circle</Icon>}
```
Comment (mddilley) [inline moped-editor/src/views/projects/Toolbar.js:40]: I was going to recommend importing like below but then I saw the `Icon` component in the MUI docs. TIL!

`import AddCircleIcon from "@material-ui/icons/AddCircle";`

`<AddCircleIcon color="primary" />`

File: moped-editor/src/views/projects/ProjectsTable.js [hunk 2]
```diff
@@ -9,27 +9,57 @@ import {
...
 
 const ProjectsTable = ({ projects }) => {
     const classes = useStyles();
 
+    /**
+     * Parses a PostgreSQL timestamp string and returns a human-readable date-time string
+     * @param {string} date - The date as provided by the database
+     * @return {string}
+     */
+    const parseDateReadable = date => {
```
Comment (mddilley) [inline moped-editor/src/views/projects/ProjectsTable.js:33]: Thanks for writing this helper! Those timestamps were hurting my eyes. 😂

**Review by mddilley — APPROVED** [review]
These improvements really sharpen up this view. ✨ I noticed that there is one unused import that is breaking the build, but, after that, good to go! 😎

File: moped-editor/src/views/projects/ProjectsTable.js [hunk 3]
```diff
@@ -9,27 +9,57 @@ import {
     TableCell,
     TableHead,
     TableRow,
-    makeStyles, Link,
+    makeStyles, Link, Chip, Icon,
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/ProjectsTable.js:12]: weird formatting happening here. I mentioned this in another PR but can you confirm that your ES Linter + Prettier is configured to run on file save Sergio?

File: moped-editor/src/views/projects/Toolbar.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ import {
     TextField,
     InputAdornment,
     SvgIcon,
-    makeStyles,
+    makeStyles, Icon,
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/Toolbar.js:12]: another formatting code smell.

File: moped-editor/src/views/projects/ProjectsTable.js [hunk 5]
```diff
@@ -9,27 +9,57 @@ import {
...
+    }
+
+    const getProjectStatus = status => {
+        const statusColorMap = {
+            "active": "primary",
+            "inprogress": "secondary",
+            "canceled": "disabled",
+        }
+
+        const statusLabel = cleanUpText(status);
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/ProjectsTable.js:52]: I really like these status labels.

**Review by mateoclarke — APPROVED** [review]
Approving but would really like to make sure code linting/formatting is consistent before and that we fix the breaking build error that @mddilley before we merge to staging.

jay_silent_bob_dancing

👍

**Pull request merged**